bootstrapper/internal/joinclient/joinclient.go

```diff
@@ -173,13 +173,13 @@ func (c *JoinClient) tryJoinWithAvailableServices() (ticket *joinproto.IssueJoin
 
 	endpoint, _, err := c.metadataAPI.GetLoadBalancerEndpoint(ctx)
 	if err != nil {
-		return nil, nil, fmt.Errorf("failed to get load balancer endpoint: %w", err)
+		c.log.Warn("Failed to get load balancer endpoint", "err", err)
```

Comment:
As far as I understand we could leave this block unchanged, because joining via node IP does not work anyway due to node-to-node strict mode. Otoh, this way it's nicely symmetric.